### PR TITLE
Add closing tag in include.

### LINKED ISSUE #260: Documentation does not close <%include> tag
**Migrated issue, originally created by Javier Ruere ([@jruere](https://github.com/jruere))**

Here: http://docs.makotemplates.org/en/latest/inheritance.html#using-include-with-template-inheritance

### PR DESCRIPTION
**Pull request by maxking**
Fixes #260

### CODE REVIEW

File: doc/build/inheritance.rst [hunk 1]
```diff
@@ -580,7 +580,7 @@ Another scenario is below, which results in both ``"SectionA"`` blocks being ren
 
     ## parent.mako
     <%inherit file="base.mako">
```
Comment (zzzeek) [inline doc/build/inheritance.rst]: need it here too

File: doc/build/inheritance.rst [hunk 2]
```diff
@@ -525,7 +525,7 @@ A common mistake is along these lines:
     </%block>
 
     ## parent.mako
-    <%include file="partials.mako">
+    <%include file="partials.mako" />
 
     ## child.mako
     <%inherit file="parent.mako">
```
Comment (zzzeek) [inline doc/build/inheritance.rst]: and here....thanks!

**Review by zzzeek — CHANGES_REQUESTED** [review]
this page still has some tags without a slash can you get those also?  thanks!

**Comment by maxking** [comment]
Hopefully I got all the ones on this page.

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **zzzeek** to try to get revision 541f219237c47ab159a7bc36172044d503cd0c08 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change 541f219237c47ab159a7bc36172044d503cd0c08: https://gerrit.sqlalchemy.org/c/sqlalchemy/mako/+/3777

**Comment by zzzeek** [comment]
looks great, thanks

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/mako/+/3777 has been **merged**. Congratulations! :)

**Pull request closed without merging**